pkgs/development/python-modules/python-fluent/fluent-pygments.nix

```
@@ -0,0 +1,24 @@
+{ buildPythonPackage, fetchPypi, lib, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "fluent.pygments";
+  version = "1.0";
+  format = "wheel";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi rec {
+    inherit pname version format;
+    sha256 = "sha256-YlyHqKI2LvMEFGsWHTWdz2Ur7Soa5IabVge44G0RfZc=";
```

The given hash is a SRI hash.

```suggestion
    hash = "sha256-YlyHqKI2LvMEFGsWHTWdz2Ur7Soa5IabVge44G0RfZc=";
```